requirements-dev.txt

```diff
@@ -41,3 +41,5 @@ einops
 transformers
 mlflow
 matplotlib
+tensorboardX
+moviepy
```

Comment:
is this required by the dev environment but not the user's?

Reply:
Hi @wyli ,

If you want to visualize video with TensorBoardX, you need to install `moviepy`, others plot with TensorBoardX don't need it. So I just added in the dev for test.
Maybe we should also add it to the general install logic `pip install monai[tensorboardX, moviepy]`.

Thanks.

Reply:
removed in the latest commit.

Thanks.